randovania/gui/lib/qt_network_client.py

```diff
@@ -157,7 +156,7 @@ async def login_with_discord(self):
 
         sid = await self.server_call("start_discord_login_flow")
         url = self.configuration["server_address"] + f"/login?sid={sid}"
-        webbrowser.open(url)
+        QtGui.QDesktopServices.openUrl(QtCore.QUrl(url))
```

Comment:
The thing returns a bool for success / failure.
I think, we should check for that and otherwise throw an error explicitly mentioning that it failed instead of silently fail like it was done and happened now.

Reply:
We could, but dunno if it's that good of an idea:
> **Warning**: A return value of `true` indicates that the application has successfully requested the operating system to open the URL in an external application. The external application may still fail to launch or fail to open the requested URL. This result will not be reported back to the application.

But i guess better to sometimes show an error, than to never show an error?

Reply:
That's preferable to returning an error when the URL was in-fact opened.